src/common/components/TypeIcon.tsx

```diff
@@ -0,0 +1,86 @@
+import { FC } from 'react';
+import { useAppSelector } from '../hooks';
+import { IconInfo } from '../slices/iconSlice';
+import { FontAwesomeIcon as FAIcon } from '@fortawesome/react-fontawesome';
+import './kbase_icons.css';
+import classes from './TypeIcon.module.scss';
+import { IconProp, library } from '@fortawesome/fontawesome-svg-core';
+import {
+  faCube,
+  faAlignJustify,
+  faTable,
+  faListUl,
+  faSliders,
+  faChartArea,
+  faTableCells,
+  faTableList,
+  faFlask,
+  faCodeFork,
+  faBullseye,
+  faArrowsLeftRight,
+  faGaugeHigh,
+  faBook,
+  faFile,
+} from '@fortawesome/free-solid-svg-icons';
+
+library.add(
```

Comment:
This step seems like it should happen in a common module shared by TypeIcon, AppCellIcon, and any other place we might use icons (LeftNavBar?). Not clear on exactly what it is doing. Does this make the icons accessible via string name?

Reply:
could maybe export a wrapped FAIcon which extends the icon prop to accept a string? (see typegaurd comment above)